lua/luasnip/loaders/from_lua.lua

```diff
@@ -34,9 +34,9 @@ local function load_files(ft, files, add_opts)
 		local func_string = path_mod.read_file(file)
 
 		local load_ok, func = pcall(loadstring, func_string)
-		if not load_ok then
+		if not load_ok or func == nil then
 			log.error("Failed to load %s\n: %s", file, func)
-			error("Failed to load " .. file .. "\n: " .. func)
+			error(string.format("Failed to load %s\n: %s", file, func))
```

Comment:
You can omit the `string.format` and just pass `file, func` to `error`, [it passes its arguments to `string.format`](https://github.com/L3MON4D3/LuaSnip/blob/45670282dc861b51c4e028e6930c75371fe0b88f/lua/luasnip/util/log.lua#L102-L106)

Reply:
Yes :+1:
I'm not saying you should not use `string.format`, I'm saying that `string.format` is called with the arguments passed to `error`, and there's no need to call that manually before passing the string to `error`.

Reply:
Aaah sorry, my bad, I thought that was a `log.error` 🤦🤦🤦